corehq/ex-submodules/pillowtop/feed/interface.py

```diff
@@ -37,6 +38,13 @@ class ChangeMeta(jsonobject.JsonObject):
     # track of retry attempts
     attempts = jsonobject.IntegerProperty(default=0)
 
+    @property
+    def producer_retry_count(self):
+        return self._producer_retries
```

Comment:
Why use a property + private attribute rather than a standard attribute? Is it so JsonObject does not save it?

Reply:
yes that's why - I could just access the private attribute directly but this seemed better (though more verbose)